tests/integration_tests/cmd/test_clean.py

```diff
@@ -38,7 +38,14 @@ class TestCleanCommand:
     )
     def test_clean_rotated_logs(self, class_client: IntegrationInstance):
         """Clean with log params alters expected files without error"""
-        assert class_client.execute("cloud-init status --wait").ok
+        # Expect warning exit code 2 on Ubuntu Noble due to cloud-init
+        # disabling /etc/apt/sources.list build artifact in favor of deb822
+        result = class_client.execute("cloud-init status --wait --long")
+        return_code = 2 if CURRENT_RELEASE.series == "noble" else 0
```

Comment:
Nit: in the future we should use something like
```python
return_code = 2 if CURRENT_RELEASE >= NOBLE else 0
```

but since this will all be reverted, it looks good as is